### ISSUE TITLE
Issues setting this up

### ISSUE THREAD

**Issue by ColdIce1** [jodal/comics#121]
I had a lot of issues settings this up. Not sure if it's still in the works...
Anyway, here is some fixes for broken initial installation on master-branch
**Use virtualenv**
1. Freeze tastypie @ django-tastypie==0.12.2
```
pip remove tastypie
pip install tastypie==0.12.2
```
2. Change some variables around that is broken in `base.py`, change the following:
```
BASE_PATH = os.path.dirname(os.path.dirname(__file__))
STATIC_ROOT = os.path.join(BASE_PATH, 'assets')
```
3. Fix the wsgi file, set the virtualenv name, e.g if your virtualenv is called `comicsenv`
```
VIRTUALENV_ROOT = os.path.join(root_path, 'comicsenv')
```

**Comment by jodal** [comment]
Thanks for the steps! I'm planning to work a bit on comics soon, to get it up and running on newer Python and Django versions. I'd also like to separate file storage enough to support e.g. Heroku+S3 or running comics in a container.

**Comment by jodal** [comment]
I think all of these issues have been addressed along the way since you filed the issue.

1. django-tastypie is currently pinned to 0.14.3, which is the latest version, and still compatible with our quite Python 2.7 and Django 1.11.
2. The settings system has been rebuilt to having a single `settings.py` and handling all variance through environment variables, ref. 52b45d223f2dbe46cf027f97f197baccce9c250f.
3. The virtualenv can now be located anywhere, as long as the environment variable `VIRTUALENV_ROOT` points to it.

All of this will soon be released as Comics 3.0. The `main` branch already reflects this, and so does the "latest" version of the docs.
